src/main/java/de/dennisguse/opentracks/content/provider/CustomContentProvider.java

```diff
@@ -356,13 +362,47 @@ private Uri insertMarker(Uri url, ContentValues contentValues) {
         throw new SQLException("Failed to insert a marker " + url);
     }
 
+    /**
+     * Return the string representing the query that compute sensor stats from trackpoints table.
+     * It computes the average for heart rate, cadence, and power (duration-based average) and the maximum for heart rate and cadence.
+     * Finally, it ignores manual pause (SEGMENT_START_MANUAL).
+     */
+    private final String getSensorStatsQuery() {
```

Comment:
Can be a property aka private final String SENSOR_STATS_QUERY = ""

Reply:
Oops!!! :see_no_evil: I read it quickly... Fixed!